docs/howtos/setup-android-build-environment.md

```diff
@@ -21,18 +21,15 @@ At the end of this process you should have the following environment variables s
 
 - `ANDROID_NDK_ROOT`
 - `ANDROID_NDK_HOME`
-- `ANDROID_NDK_TOOLCHAIN_DIR`
 - `ANDROID_NDK_API_VERSION`
 - `ANDROID_HOME`
 - `JAVA_HOME`
 
 These variables are required every time you build, so you should add them to
 a rc file or similar so they persist between reboots etc.
 
-1. Install NDK r15c from https://developer.android.com/ndk/downloads/older_releases
-   (yes, this sucks, but newer versions don't understand the `--deprecated-headers`
-   argument required to build OpenSSL against a v21 toolchain).
-    - Extract it, put it somewhere (`$HOME/.android-ndk-r15c` is a reasonable
+1. Install NDK r20 from https://developer.android.com/ndk/downloads/older_releases
+    - Extract it, put it somewhere (`$HOME/.android-ndk-r20` is a reasonable
       choice, but it doesn't matter), and set `ANDROID_NDK_ROOT` to this location.
     - Set `ANDROID_NDK_HOME` to match `ANDROID_NDK_ROOT`, for compatibility with
```

Comment:
Do we still need NDK_HOME as well?

Reply:
Yup `ANDROID_NDK_HOME` is still needed by gradle I believe.